test/TestAssets/SimpleDataCollector/Class1.cs

```diff
@@ -40,7 +40,7 @@ public void TestSessionStart(TestSessionStartArgs testSessionStartArgs)
             Console.WriteLine(testSessionStartArgs.Configuration);
             File.WriteAllText(this.fileName, "TestSessionStart : " + testSessionStartArgs.Configuration + "\r\n");
 #if NETFRAMEWORK
-            var appDomainFilePath = Path.Combine(Path.GetTempPath(), "appdomain_datacollector.txt");
+            var appDomainFilePath = Environment.GetEnvironmentVariable("AppDomainDataCollectorFilePath");
```

Comment:
This should probably be consistent with `Environment.GetEnvironmentVariable("SampleDataCollectorTempPath") ?? Path.GetTempPath();` and use the temp path as fallback. This would help when you have to debug the test itself in VS.

Reply:
I am actually wondering whether we should fail when the path is not set?

Reply:
I was considering that as well, but if the test fails for some weird reason (e.g. it seems like the task that the test should do did not happen at all), it's easier to just open the test solution and run the test there interactively to debug it. In such case the value would be empty and this would throw.